docs/Documentation/eagle_to_kestrel_transition.md

```diff
@@ -0,0 +1,144 @@
+
+**NOTE**: Some MPI codes, especially old legacy scientific software, may be difficult to build with Cray MPICH. In these cases, if it is possible to build the code with Intel MPI or a different MPICH implementation, then Cray MPICH can be utilized at run-time via use of the `cray-mpich-abi` module (note that OpenMPI is *NOT* an implementation of MPICH, and you cannot use the `cray-mpich-abi` if you built with OpenMPI). A detailed example of building with Intel MPI but running with Cray MPICH can be found on our [VASP application page](/Documentation/Applications/vasp/). 
+
+## 4. Running your jobs on Kestrel
+
+See our page on submitting jobs on Kestrel [here](/Documentation/Systems/Kestrel/running/).
+
+Submitting a job on Kestrel works much the same as submitting a job on Eagle. Both systems use the Slurm scheduler. If the application you wish to run can be found under our [Applications tab](/Documentation/Applications/), then there may be example Kestrel submit scripts on the application page. Otherwise, our [VASP documentation page](/Documentation/Applications/vasp/#vasp-on-kestrel) contains a variety of sample submit scripts that you can modify to fit your own purposes.
+
+### Shared Partition
+
+Note that each Kestrel CPU node contains 104 CPU cores (and 256 GB memory). Some applications or application use-cases may not scale well to this many CPU cores. In these cases, it is recommended to submit your jobs to the shared partition. A job submitted to the shared partition  will be charged AUs proportionate to whichever resource you require more of, between CPUs and memory.
+
+The following is an example shared-partition submit script using VASP:
+
+```
+#!/bin/bash
+#SBATCH --nodes=1
+#SBATCH --partition=shared
+#SBATCH --tasks=26 #How many cpus you want
+#SBATCH --mem-per-cpu=2G #Default is 1 GB/core but 2 GB/core is a good starting place for VASP
+#SBATCH --time=2:00:00
+#SBATCH --account=<your-account-name>
+#SBATCH --job-name=<your-job-name>
+
+module load vasp/<version>
+
+srun vasp_std |& tee out
+```
+
+For example, if you request 52 CPU cores (half of the available CPU cores), but 256 GB memory (all of the memory), you will be charged the AU-equivalent of a full node. If you request 26 cores and 2gb per core, you will be charged 0.25*10 AUs = 2.5 AUs per hour, because:
+
+2 gb mem per cpu\*26 cpus = 52 gb memory, and 52/256 = 0.2, or 20% of the memory of the node.
+
+26 CPU cores / 104 CPU cores = 0.25, or 25% the CPU cores of the node.
+
+In this example, you are requesting 20% of the memory of a node, but 25% the available CPUS of the node, and are therefore charged 25% of a regular AU.
+
+For more information on the shared partitions, see [here](/Documentation/Systems/Kestrel/running/#shared-node-partition).
+
+For more information on Kestrel's overall configuration, see [here](https://nrel.github.io/HPC/Documentation/Systems/Kestrel/running/#partitions).
```

Comment:
This might be beneficial to link to: https://www.nrel.gov/hpc/kestrel-system-configuration.html